apps/server/src/app/app.ts

```diff
@@ -148,6 +149,7 @@ app.use('/tools', devOnly, toolsRouter)
 app.use('/v1', webhooksRouter)
 
 app.use('/v1', publicRouter)
+app.use('/v1/auth-users', authUserRouter)
```

Comment:
Need to figure out a better authorization pattern for this route / refactor checking for existing user by email and creating new users so those routes aren't open to the wild.